Codeinwp/neve
# Development

### Summary
- fixed issue with scrolling disabled on mobile for menu anchors
- removed list style global reset that was affecting elementor
- removed css rule that hides featured image for Neve 2018 starter site
- fixed errors thrown on lower than 5.3.29 PHP Versions
- trying to add back PHPUnit tests
- change travis process to build stages and check code style on single container


### Will affect visual aspect of the product

NO

### Test instructions


- Test anchors in menu that point to the same page (on mobile), the scroll should work fine after you click the menu item
-  Check `<ul>` style in elementor - it should look ok (more details in #342)
- Import `Neve 2018` - the featured images on blog should appear
- Check if PHPUnit tests work properly on travis


Closes #341, #342, #331
